pyrolite/mineral/normative.py

```diff
@@ -1095,9 +1105,9 @@ def corr_m_wt(oxide):
     df["deficit"] = df["D2"] > 0
 
     # Normative Nepheline / Albite
-    df["Ne_"] = np.where((df["D2"] < 4 * df["Ab_p"]), df["D2"] / 4, df["Ab_p"]).T
+    df["Ne_"] = np.where(df["D2"] < 4 * df["Ab_p"], df["D2"] / 4, df["Ab_p"]).T
```

Comment:
With the updates to parentheses elsewhere, it might be good to also add some here, for readability `df["D2"] < (4 * df["Ab_p"])`, but the order of operations is the same either way.